torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -209,6 +213,12 @@ struct GraphFuser {
       return false;
     if (!node->is_constant(attr::dim))
       return false;
+
+    Node* list_construct = node->namedInput(attr::tensors)->node();
```

Comment:
`list_construct` is the same as `tensors_node` a few lines below, you can move `tensors_node` line here and reuse it.

Reply:
Done. Will push a commit soon after testing. Thanks.